examples/minimal/providers.tf

```diff
@@ -1,4 +1,4 @@
 provider "azurerm" {
-  version = "=2.11.0"
+  version = "=2.60.0"
   features {}
```

Comment:
Not related, but we should probably include our partner id in the examples.

https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs#partner_id